admin/templates/resource/custom-list.html

```diff
@@ -421,5 +458,37 @@ <h3 class="mb-6 text-2xl font-medium text-gray-900">Edit History</h3>
             console.error('Error:', error);
         });
     }
+
+    function toggleDropdown(id, buttonId) {
+        const dropdown = document.getElementById(id);
+        dropdown.classList.toggle('hidden');
+    }
+
+    function selectOption(optionText, mandiValue, cropValue, buttonLabelId, dropdownId) {
+        const buttonLabel = document.getElementById(buttonLabelId);
+        buttonLabel.textContent = optionText;
+        if (mandiValue) {
+            selectedMandi = mandiValue;
+        }
+        if (cropValue) {
+            selectedCrop = cropValue;
+        }
+
+        const dropdown = document.getElementById(dropdownId);
+        dropdown.classList.add('hidden');
+        let pendingButton = document.getElementById('pendingButton');
+        let allReceiptButton = document.getElementById('allReceiptButton');
+        let resourceType = "{{ resource_type }}";
+
+        buttonValue = window.location.href.indexOf("pending") > -1?"pending":"all"
+        let url = `/admin/resource/${resourceType}/${buttonValue}?mandi=${selectedMandi}&crop=${selectedCrop}`;
+        window.location.href = url;
```

Comment:
since you are reloading the page again, will the user see their selected filters in mandi and crop values when they visit the page with `/admin/resource/mandi-receipt/pending/?mandi=1&crop=2` or something similar link?

If not, we should show the filters applied as pre-selected in the mandi & crop dropdowns.

Reply:
link is working but filter applied as mandi and crop not get pre-selected in dropdowns, I'm stuck for  this UI fix.